sensor/common/chaos/env.go

```diff
@@ -0,0 +1,27 @@
+package chaos
+
+import (
+	"strings"
+
+	"github.com/stackrox/rox/pkg/env"
+)
+
+var (
+	centralEndpointNoProxyEnv = env.RegisterSetting("ROX_CENTRAL_ENDPOINT_NO_PROXY")
+	chaosProxyEnabledEnv      = env.RegisterBooleanSetting("ROX_CHAOS_PROXY_ENABLED", false)
+	chaosProfileEnv           = env.RegisterSetting("ROX_CHAOS_PROFILE")
```

Comment:
Are both these required or could we just check if profile is empty?

Reply:
Done. I ended up adding a new profile `'none'` because I still want to have the possibility to deploy the chaos proxy without any network condition preconfigured. This allows manually switching some condition on/off. 

So it ended up being:

**Chaos profile settings:**
`""`: Disabled
`"none"`: Enabled without any network condition
`"<PROFILE>"`: Enabled with network condition defined in `sensor/common/chaos/profile/<PROFILE>.go`